.github/workflows/regression-report.yml

```diff
@@ -454,10 +454,30 @@ jobs:
 
       - name: Prepare environment
         run: |
-          mv .ci-temp/project.properties ./.ci-temp/contribution/checkstyle-tester/
-          mv .ci-temp/*.xml ./.ci-temp/contribution/checkstyle-tester/
+          mv .ci-temp/project.properties ./.ci-temp/test-configs/checkstyle-tester/
+          mv .ci-temp/*.xml ./.ci-temp/test-configs/checkstyle-tester/
           sudo apt install groovy
```

Comment:
Please remove this line , to be sure we do not use groovy.
Please Rename step, as it's goal becomes different

Reply:
done